compiler/noirc_frontend/src/hir/mod.rs

```diff
@@ -180,6 +180,20 @@ impl Context {
             .collect()
     }
 
+    /// Returns the [Location] of the definition of the given Ident found at [Span]
+    /// Return [None] when definition is not found.
+    pub fn find_definition_location(
+        &self,
+        file: FileId,
+        query_location: &Span,
```

Comment:
just wondering: do you think its possible for this method to take in a DefinitionId? -- If so, we can open up an issue for this, it would mean that the method would probably become:

```
fn find_definition_location(&self, id : DefinitionId) -> Location{}
```

Reply:
DefinitionId does not carry location data we are concerned aboout to get definition coming from File+Line+Column